template/v2/Dockerfile

```diff
@@ -91,6 +92,11 @@ RUN if [[ -z $ARG_BASED_ENV_IN_FILENAME ]] ; \
     fi && \
     # Enforce dependencies are all installed from conda-forge
     micromamba install -y --name base --file /tmp/$ENV_IN_FILENAME && \
+    JUPYTERLAB_VERSION=$(grep "^conda-forge::jupyterlab" /tmp/$ENV_IN_FILENAME) && \
```

Comment:
When I ran this locally, it also picked up other packages starting with jupyterlab e.g.
```
$ echo $JUPYTERLAB_VERSION                                                                                                            
conda-forge::jupyterlab[version='>=4.2.5,<4.3.0']
conda-forge::jupyterlab-lsp[version='>=5.0.3,<5.1.0']
conda-forge::jupyterlab-git[version='>=0.50.2,<0.51.0']
```
Might need something like `grep "^conda-forge::jupyterlab\["` to get just actual jupyterlab package